### ISSUE TITLE
License of project

### ISSUE THREAD

**Issue by victor4pinheiro** [Avengers042/petshop-frontend#4]
What's a good license to put in this project? Being developer, I want the modifications back to this repo because it can benefit everyone who can use this repo.

My suggestions is that we should use GPL3, because it can provide more compatibility with more Open Source licenses and it's more strict (I really like the the concept of Free Software), but it can provide some problems in some situations.

What's your thoughts, @wesleyclaudino ?

Links:

Free Software: https://www.gnu.org/philosophy/free-sw.html
Licenses: https://www.gnu.org/licenses/license-list.html

**Comment by wesleyclaudino** [comment]
In this part of the license (GPL3), it would not be necessary to do this in our case since the school will not be using the project, but will only use it for evaluation, right?

```You should also get your employer (if you work as a programmer) or school, if any, to sign a “copyright disclaimer” for the program, if necessary. For more information on this, and how to apply and follow the GNU GPL, see <https://www.gnu.org/licenses/>.```

**Comment by victor4pinheiro** [comment]
Yes, it's not necessary, but we need to pay attention, because when we send the project we must define the license in AVA as free (i don't remember for now what license are in AVA).

**Comment by wesleyclaudino** [comment]
So it would be necessary to change the licenses in the middle of the project? And then we would have to go back to the previous license again? Bearing in mind that the project will be available to the MEC after submission to the AVA, we have to see how is the possibility of changing the license after this submission is made, perhaps this could cause problems such as infringing some points of the GPL3.

**Comment by victor4pinheiro** [comment]
Seeing the AVA:

![image](https://user-images.githubusercontent.com/60495567/235917908-f8a5e641-3080-4f3f-be6e-bd35d46945eb.png)

If we choose "Outro" (Other), we can choose GPL3. 

Answering your question: It's not necessary to change anything, but using MIT is not so beneficial to ecosystem as using GPL, because if anyone use a project with MIT, they don't need to open-source the modifications, i.e., they can privatize the modifications and this is a thing I really don't like.

**Comment by wesleyclaudino** [comment]
Yes, I didn't even think about using the MIT license, since we can send it to the AVA without having to make that change, perfect. Let's go with the GPL3 license.
